schema/bom-1.6.xsd

```diff
@@ -3237,6 +3244,333 @@ limitations under the License.
         </xs:attribute>
     </xs:complexType>
 
+    <xs:complexType name="environmentalConsiderationsType">
+        <xs:annotation>
+            <xs:documentation>
+                Describes various environmental impact metrics.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="energyConsumptions" type="bom:energyConsumptionType" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation>
+                        Describes energy consumption information incurred for one or more component lifecycle activities.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="energyConsumptionType">
```

Comment:
the values are very much ML-specific.

please prefix the element name with something appropriate. like `mlEnergyConsumptionType`

Reply:
@jkowalleck we should not use `mlEnergyConsumptionType` if this is to be used elsewhere. I agree with the current name of `energyConsumptionType`.